source/editableText.py

```diff
@@ -249,11 +250,20 @@ def detectPossibleSelectionChange(self):
 		self._lastSelectionPos=newInfo.copy()
 		if not oldInfo:
 			# There's nothing we can do, but at least the last selection will be right next time.
+			self.isTextSelectionAnchoredAtStart=True
 			return
+		self._findSelectionAnchor(oldInfo,newInfo)
 		hasContentChanged=getattr(self,'hasContentChangedSinceLastSelection',False)
 		self.hasContentChangedSinceLastSelection=False
 		speech.speakSelectionChange(oldInfo,newInfo,generalize=hasContentChanged)
 
+	def _findSelectionAnchor(self,oldInfo,newInfo):

```

Comment:
There's a slight chance someone might see this method name and expect it to return something, rather than taking action. I think renaming this to `_updateSelectionAnchor` or similar might make this clearer.